content/applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst

```diff
@@ -0,0 +1,193 @@
+:guilabel:`Integration`. The naming convention is as follows: `Odoo CompanyName`, with the user's
+company name replacing `CompanyName`.
+
+Then, check the box next to :guilabel:`Service Points` and select the shipping services for this
+integration. There is no need to enter a :guilabel:`Webhook URL`. After saving, the
+:guilabel:`Public and Secret Keys` are generated.
+
+.. image:: sendcloud_shipping/public-secret-keys.png
+   :align: center
+   :alt: Configuring the Sendcloud integration and receiving the credentials.
+
+Setup in Odoo
+=============
+
+Install the Sendcloud shipping module
+-------------------------------------
+
+After the Sendcloud account is set up and configured, the user can configure their Odoo database.
+To get started, go to Odoo's :guilabel:`Apps` module and search for the :guilabel:`Sendcloud
+Shipping` module. Then, install this module, if it is not already installed.
+
+.. image:: sendcloud_shipping/sendcloud-mod.png
+   :align: center
+   :alt: Sendcloud Shipping module in the Odoo Apps module.
+
+Sendcloud shipping connector configuration
+------------------------------------------
+
+Ensure the :guilabel:`Sendcloud Shipping Module` is activated in :menuselection:`Inventory -->
+Configuration --> Settings`. The :guilabel:`Sendcloud Connector` setting is found under the
+:guilabel:`Shipping Connectors` section.
+
+After activating the :guilabel:`Sendcloud Connector`, click on the :guilabel:`Sendcloud Shipping
+Methods` link below the listed connector. Once on the :guilabel:`Shipping Methods` page, click
+:guilabel:`Create`.
+
+.. tip::
+   :guilabel:`Shipping Methods` can also be accessed by going to :menuselection:`Inventory -->
+   Configuration --> Delivery --> Shipping Methods`.
+
+Fill out the following fields in the :guilabel:`New Shipping Method` form:
+
+- :guilabel:`Shipping Method`: type `Sendcloud DPD`.
+- :guilabel:`Provider`: select :guilabel:`Sendcloud` from the drop-down menu.
+- :guilabel:`Delivery Product`: set the product that was configured for this shipping method or
+  create a new product.
+- In the :guilabel:`SendCloud Configuration` tab, enter the :guilabel:`Sendcloud Public Key`.
+- In the :guilabel:`SendCloud Configuration` tab, enter the :guilabel:`Sendcloud Secret Key`.
+- Manually :guilabel:`Save` the form by clicking the cloud icon next to the :guilabel:`Shipping
+  Methods / New` breadcrumbs.
+
+After configuring and saving the form, follow these steps to load the shipping products:
+
+- In the :guilabel:`SendCloud Configuration` tab of the :guilabel:`New Shipping Method` form, click
+  on the :guilabel:`Load your SendCloud shipping products` link.
+- Select the shipping products the company would like to use for deliveries and returns.
+- Click :guilabel:`Select`.
+
+For example:
+
+.. image:: sendcloud_shipping/sendcloud-example.png
+   :align: center
+   :alt: Example of shipping products configured in Odoo.
```

Comment:
Same here.
Break the relationship between the text and actually write out the example, using an example block.